css/style.css

```diff
@@ -0,0 +1,73 @@
+*{
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+/* body{
+    display: block;
+} */
+
+#navbar{
+    /* border: 2px solid red; */
+    position: fixed;
+    left: 0;
+    display: flex;
+    flex-direction: column;
+    height: 100%;
+    min-width: 250px;
+    overflow: auto;
+    border-right: 2px solid black;
+    background-color: white;
+}
+
+header{
+    padding: 0.5em 0 0.5em 1em;
+}
+
+.nav-link{
+    text-decoration: none;
+    color: black;
+    text-align: left;
+    border-top: 2px solid black;
+    padding: 0.5em 0 0.5em 1em;
+    margin-top: 0.5em;
+}
+
+main{
+    margin-left: 260px;
+}
+

```

Comment:
![marginErr](https://user-images.githubusercontent.com/68627859/235577324-e4494eb4-15ef-45ea-a390-b86eb1e389d2.png)
### Margin problem
- [ ]  Please fix the margin error :smile_cat: :